arcgis-ios-sdk-samples/Route & Directions/Find closest facility to an incident (interactive)/README.metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "category": "Routing and Directions",
+    "category": "Route and Directions",
```

Comment:
Route and directions (lower case d)

Reply:
Agreed to leave as Route and Directions. The casing will be addresses as a cross-platform issue.